### PR TITLE
Nerfs buckshot

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Buckshot now does a maximum of 75 damage, from 90.
/:cl:

Welcome to the balance discussion!
I'd be nerfing this more, but small steps while I berate anyone and everything for thinking this is wise.

1. Pixel projectiles made spread weapons much more reliable, as they don't have a randomized spread anymore; if you fire the same gun in the same direction the spread is always the same.
2. With spread weapons being more reliable, the issues with jacking the damage up compared to single-projectile stuff becomes more apparent; spread is supposed to trade spikes of damage at range for reliability.
3. At all but the most extreme ranges you'd be hit with at least two pellets, which would do half the damage of a shotgun slug. This is quite clearly *too reliable*.
4. At the same range, the shotgun slug has about a 35% chance to hit(ie; does 35% damage, 21 (this is inaccurate, as it either hits or doesn't, but it gets the meaning across)) and is accordingly less effective both at long range and up close.

### CODE REVIEW

**Comment by Arianya** [comment]
Out of interest, why touch max damage instead of touching down the number of pellets? This would make point blank shots with buckshot still scary, but lessen the effect noted in non-blank firing ranges.

**Comment by ChangelingRain** [comment]
Tweaking the number of pellets affects the spread and makes it a tighter cone.

**Comment by Iamgoofball** [comment]
feature freeze officially over

**Comment by Kromgar** [comment]
Finally shotties need to be nerfed they are now THE weapon of choice everyone loves the fucking things.

**Comment by WJohn** [comment]
So as I see it, this nerf is required because buckshot shouldn't function so much of a direct damage upgrade (which it is, if not equal to the slug) as it is a bullet type where at least SOME damage is basically guaranteed. Slugs put all their eggs in one basket for 60 brute, shells should do 60-75 brute if all pellets hit (which is unlikely), but you're more or less guaranteed to do at least 30-45.

**Comment by KingPhilipIII** [comment]
Fucking finally. I hate the overuse shotguns see because it's so easy to get destroyed by one. I'm still salty as fuck over the one time I'm in a cohesive nuke op unit(Advancing as a team with two people using swords and shields as a vanguard while gunners and medics bring up the rear) and the team gets decimated by the AI closing fire doors behind us and someone off screen spamming buckshot until we're all dead.

I understand that a few overpowered weapons is no issue but few are as abused as the shotgun to cheese antagonists.
Slaughter demons? Forget about cleaning up. Just stand guard with a shotgun, run up and kill it in one shot at point blank.
 Lesser ashdrake? Buckshot it to death with one person evading the raining fire in four or five shots.  
Any slow moving opponent? Apply buckshot to face one time and then smack it a couple times while it bleeds to death on the ground without ever getting to fight back because it's like an instant stun except it kills you at the same time.  

God this triggers me. I'd rather see bigger nerfs but atleast this problem is finally being addressed.

**Comment by Shapsy** [comment]
Shotguns are already bulky and their shells take up a lot of resources to make, space in bags to stockpile, and time to print them off and manually reload them in the gun, you guys are overexaggerating

**Comment by KorPhaeron** [comment]
Stop opening balance PRs

**Comment by KorPhaeron** [comment]
@shapsy goof and coreoverload broke cargo so that you can get enough points to arm an 80 person crew with them though

**Comment by SweaterKittens** [comment]
Not to derail this with a discussion about Cargo, but I agree with Kor. No round over 20 minutes where the station is not immediately destroyed end up without cargo absolutely FILLED with crates of anything and everything. Every time a syndie cargo tech gets in there, there's basically guaranteed to be a shuttle filled to the brim with syndicate gear crates. The 'difficulty' of getting them and the extra work it takes to load and cock them are minimal in comparison to the immense damage output.

For the record, I don't think shotguns are that overpowered, and I think there are much more unbalanced things (specifically lavaland rewards) but I think this is a reasonable balance change that will bring shotguns into line with the other guns, and return them to a comfortable place as a reliable close-quarters weapon.

**Comment by Robustin** [comment]
Shotguns are definitely, innately, overpowered. Making them harder to get is not a complete solution.

I have no bias on this one, I've been on both ends of the shotgun blasting. The issue is that they are just absurdly robust. A single laser does 20 damage, buckshot fires 6 pellets of 15 damage goodness at the same rate of fire as a laser and the spread is PERFECT with the projectile refactor. The spread is fucking godly now considering that 90% of fights take places in space with limited space to maneuver perpendicular to your shooter. Not as much of as issue when you only have to dodge one projectile at a time but with 6x spreading evenly across 3 tiles as they move down the hall, its game over if someone opens up on you from anything less than the edge of your screen. 

The balancing of shotguns should be a simple inquiry, are shotguns still be the weapon at 1-3 tile range... if the answer is yes, then keep nerfing it until the answer is "maybe, maybe not" then buff it back a little. On top of that, make it so that cargo can't order infinity of them and bust them open with minimal effort. 

Since the pixel projectile thing my biggest issue is how effective spraying weapons has become. Between disablers "lol it looks like its going to the tile to your right but the pixels are a mirage and it actually hit you dead center" and shotguns, its become way too effective to just spray at people in any situation because once 2 disablers or 3-4 pellets connect, its game over. 

Making disabler beams easier to anticipate and making shotguns less of a meme can also be combined with revisiting ebows or other traitor memes so that the station doesnt get unilaterally fucked by weapon nerfs. I see issues on both sides but sitting high and far above the rest is shotgun spam being the best answer to absolutely any combat situation.

**Comment by dannno** [comment]
tl;dr but to anyone who might be against a shotgun nerf;
A man standing 4~ish tiles away from another man can shoot-pump-shoot and instantly kill that man in 1 actual second

**Comment by Iamgoofball** [comment]
What if we coded damage fallout

**Comment by Robustin** [comment]
It'd be easier to just reduce damage, widen spread significantly. That was it can still fill the close quarters role without letting you crit people on the edge of your screen faster than a fucking laser rifle would.

**Comment by anconfuzedrock** [comment]
I don't think shorts are op I think everything else is up
